### ISSUE TITLE
Create Footer component

### ISSUE THREAD

**Issue by comrade-kirby** [HiraethOctoberStrike/octoberstrike.com#15]
blocked by #16 

![image](https://user-images.githubusercontent.com/19193721/127042763-7dd745a9-e213-4772-a43e-28ab949c3d11.png)

[Figma](https://www.figma.com/file/zBXRu0Gb5WquL5FXddFG1u/strike?node-id=0%3A1)

- add placeholder for logo (actual image added later in a separate issue)
- layout and add images for social links (correct links added later)
- add links to about, get involved, FAQ, and strike with US
